docs/design/pod-scheduling-readiness.md

```diff
@@ -0,0 +1,29 @@
+# Pod Scheduling Readiness
+## Motivation
+
+Pod Scheduling Readiness is a beta feature in Kubernetes v1.27. Users expect Volcano to be aware of it. By specifying/removing a Pod's `.spec.schedulingGates`, which is an array of strings, users can control when a Pod is ready to be considered for scheduling. For Pods with none-empty `schedulingGates`, it will only be removed by kube-scheduler once all the gates are removed.
```

Comment:
>it will only be removed by kube-scheduler once all the gates are removed.

Actually the `schedulingGates` field is usually removed by external controllers and kube-scheduler just check this filed to determine whether to schedule.